docs/Files/UI/Solution-Manager/Library/Logs/List-Log-Files.md

```diff
@@ -0,0 +1,11 @@
+# List Log Files
+
+The **Logs** module allows you to view list of current log files.
+
+![A screen showing list of log files](../../../../../Resources/Images/SM/Library/Logs/Logs-List-Log-Files.png "List of Log Files")
+
+### Filtering & Sorting
+You can filter and sort log files by using the column headers in the list.
+
+### Log File Information
+Mouse over the ![Info icon](../../../../../Resources/Images/SM/Library/Logs/Logs-Info-Icon.png "Log File Description") icon to see more information about the log file.
```

Comment:
This can stay for now, but the info icon will (should) go away and the users should be able to click a row and open the drawer with info including the description.